From akpm@osdl.org Fri Jul  7 00:11:48 2006
Message-Id: <200607070711.k677BjPv031223@shell0.pdx.osdl.net>
Subject: USB: kill usb kconfig warning
To: greg@kroah.com
Cc: akpm@osdl.org
From: akpm@osdl.org
Date: Fri, 07 Jul 2006 00:11:45 -0700

From: Andrew Morton <akpm@osdl.org>

drivers/usb/host/Kconfig:87:warning: 'select' used by config symbol 'USB_OHCI_HCD' refer to undefined symbol 'I2C_PNX'


Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/Kconfig |    1 -
 1 file changed, 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/host/Kconfig
+++ gregkh-2.6/drivers/usb/host/Kconfig
@@ -84,7 +84,6 @@ config USB_OHCI_HCD
 	depends on USB && USB_ARCH_HAS_OHCI
 	select ISP1301_OMAP if MACH_OMAP_H2 || MACH_OMAP_H3
 	select I2C if ARCH_PNX4008
-	select I2C_PNX if ARCH_PNX4008
 	---help---
 	  The Open Host Controller Interface (OHCI) is a standard for accessing
 	  USB 1.1 host controller hardware.  It does more in hardware than Intel's
